packages/core-utils/src/route.js

```diff
@@ -261,11 +381,14 @@ function makeMultiCriteriaSort(...criteria) {
  *    those with shortNames.
  *  6. longName as string.
  */
-export const routeComparator = makeMultiCriteriaSort(
-  makeNumericValueComparator(obj => getRouteSortOrderValue(obj.sortOrder)),
-  routeTypeComparator,
-  alphabeticShortNameComparator,
-  makeNumericValueComparator(obj => parseInt(obj.shortName, 10)),
-  makeStringValueComparator(obj => obj.shortName),
-  makeStringValueComparator(obj => obj.longName)
-);
+export function makeRouteComparator(transitOperators) {
```

Comment:
I'm not sure the previous jsdoc exactly matches this new function.

Reply:
Fixed in 5163a219c2ab5072d9338041435293fda8877c42